token-minter/sources/modules/mint_stage.move

```diff
@@ -37,27 +35,29 @@ module minter::mint_stage {
     /// The amount of tokens should be greater than zero.
     const EINSUFFICIENT_AMOUNT: u64 = 11;
 
-
     #[resource_group_member(group = aptos_framework::object::ObjectGroup)]
     struct MintStageData has key {
-        /// stage ("private", "presale", "public", etc.) to mint stage mapping.
-        mint_stages: SimpleMap<String, MintStage>,
-        /// Stages sorted from the earliest start time.
-        stages: vector<String>,
+        mint_stages: vector<MintStage>,
     }
 
+    /// stage ("private", "presale", "public", etc.) to mint stage mapping.
```

Comment:
1. If two stages got added with the same start time, the first one that got added will be earlier in the vector.
2. No, you can only create and remove mint stages.
3. Then this will not be executed. We give an option to the user whether to execute active stages or not

Reply:
The one that got added first will be executed first.